docs/pages/release_notes_old.md

```diff
@@ -2163,7 +2163,7 @@ More information is available in [the user documentation](pmd_userdocs_cpd.html#
     *   [#1405](https://github.com/pmd/pmd/issues/1405): \[java] New Rule: UseTryWithResources - Replace close and IOUtils.closeQuietly with try-with-resources
     *   [#1555](https://github.com/pmd/pmd/issues/1555): \[java] UnusedImports false positive for method parameter type in @see Javadoc
 *   java-codestyle
-    *   [#1543](https://github.com/pmd/pmd/issues/1543): \[java] LinguisticNaming should ignore overriden methods
+    *   [#1543](https://github.com/pmd/pmd/issues/1543): \[java] LinguisticNaming should ignore overridden methods
```

Comment:
I don't think it's useful to fix things in this file. Those release notes are old. The original version with typos is probably still somewhere on the release page, and the PRs also have typos in their name. I'd say, leave them be

Reply:
@oowekyala done.